Source/Tests/MonoGame.Extended.Tests/Collections/BitVector32Tests.cs

```diff
@@ -171,22 +172,22 @@ public void SectionIncorrectSize()
         [Fact]
         public void NegativeIndexer()
         {
-            var bitVector = new BitVector32(uint.MaxValue);
-            Assert.True(bitVector[uint.MinValue], "UInt32.MinValue");
+            var bitVector = new BitVector32(-1);
+            Assert.True(bitVector[0], "Int32.MinValue");
         }
 
         [Fact]
         public void TestSectionIndexer()
         {
-            var bitVector = new BitVector32(uint.MaxValue);
+            var bitVector = new BitVector32(-1);
```

Comment:
-1 is the same as MaxValue?

Reply:
Since we are using `int` instead of `uint` we have [Two's compliment](https://en.wikipedia.org/wiki/Two%27s_complement) to deal with. The intent of the original code was for all 32 bits to be 1's, so -1 is opposite to 0.

Reply:
All 32 bits being 1's is equal to int.MinValue - negative one is 0b1000....0001

Reply:
`int.MinValue` failed the test. `int.MinValue` resulted in `10000000_00000000_00000000_00000000`